metabase/metabase
# Add approved domains for notifications setting

### Description

[Product doc](https://www.notion.so/metabase/Advanced-Subscription-and-Alert-Controls-af55b14cd4b849aeb2d05a361674ce91)

Adds a new setting for approved domains for notifications.

### How to verify
- Merge https://github.com/metabase/metabase/pull/17897
- Run Metabase EE
- Open Admin > Settings > General
- Set a value for `APPROVED DOMAINS FOR NOTIFICATIONS`
- Ensure notifications can be sent only to approved domains

[Local email setup doc](https://github.com/metabase/metabase/wiki/Setting-up-email-in-development)

### Screenshot
<img width="495" alt="Screen Shot 2021-08-31 at 15 53 16" src="https://user-images.githubusercontent.com/14301985/131505869-ab153926-62e6-4814-95dc-9c883e23b16e.png">
